backend/configs/main.conf

```diff
@@ -1 +1,5 @@
-ipadress=127.0.0.1
\ No newline at end of file
+ipadress=127001
```

Comment:
Its just a placeholder since we have no real camera's to use anyway. Since the fix for the configreader is nog yet in this branch we cannot use dots.